src/notation/view/widgets/editstyle.cpp

```diff
@@ -86,6 +86,7 @@ static const QStringList ALL_PAGE_CODES {
     "staff-text",
     "tempo-text",
     "lyrics",
+    "expression",
```

Comment:
It looks like this might resolve #19203. That would be worth trying out!

Reply:
Yeah it should do!